src/docs/product/error-monitoring/filtering/index.mdx

```diff
@@ -23,3 +25,35 @@ Inbound filters include:
 Once applied, you can track the filtered events (numbers and cause) using the graph provided at the top of the Inbound Data Filters view.
 
 ![](builtin-inbound-filters.png)
+
+## Filtering for Session Events
+
+Data filters are only partially supported for session events, and they work slightly differently:
+
+### Equivalent Filters
```

Comment:
I checked again and I'm afraid that filters do not apply to sessions at all. We will revisit this in the future, so lets just state that inbound data filters do not impact sessions.

At the same time, _most_ filters will apply to transactions just like error events. That is, as long as they make sense. The one exception is Error Message filters. They obviously cannot apply to transactions since those do not have an error message.

Reply:
I pulled this content from the Release Health page content; I will fix it so it is corrected.

Reply:
I've removed it from the content, and stored it in Notion once we determine where to publish it.